nservicebus/bridge/configuration_failed-messages_bridge_[2,).partial.md

```diff
@@ -0,0 +1,16 @@
+### Handling of failed messages
+
+By default, when the bridge transfers a message to the ServiceControl transport it will not attempt to translate the [`NServiceBus.ReplyToAddress`](/nservicebus/messaging/headers.md#messaging-interaction-headers-nservicebus-replytoaddress) message header.
```

Comment:
```suggestion
By default, when the bridge transfers a message to the ServiceControl error queue it will not attempt to translate the [`NServiceBus.ReplyToAddress`](/nservicebus/messaging/headers.md#messaging-interaction-headers-nservicebus-replytoaddress) message header.
```